python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1224
   - name: u_kronFlux_flag_bad_radius
     "@id": "#Object.u_kronFlux_flag_bad_radius"
     datatype: boolean
     description: Bad Kron radius. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1225
   - name: u_kronFlux_flag_bad_shape
     "@id": "#Object.u_kronFlux_flag_bad_shape"
     datatype: boolean
     description: Shape for measuring Kron radius is bad; used PSF shape. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1226
   - name: u_kronFlux_flag_bad_shape_no_psf
     "@id": "#Object.u_kronFlux_flag_bad_shape_no_psf"
     datatype: boolean
     description: Bad shape and no PSF. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1227
   - name: u_kronFlux_flag_edge
     "@id": "#Object.u_kronFlux_flag_edge"
     datatype: boolean
     description: Bad measurement due to image edge. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1228
   - name: u_kronFlux_flag_no_fallback_radius
     "@id": "#Object.u_kronFlux_flag_no_fallback_radius"
     datatype: boolean
     description: No minimum radius and no PSF provided. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1229
   - name: u_kronFlux_flag_no_minimum_radius
     "@id": "#Object.u_kronFlux_flag_no_minimum_radius"
     datatype: boolean
     description: Minimum radius could not be enforced, no minimum value or PSF. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1230
   - name: u_kronFlux_flag_small_radius
     "@id": "#Object.u_kronFlux_flag_small_radius"
     datatype: boolean
     description: Measured Kron radius was smaller than that of the PSF. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1231
   - name: u_kronFlux_flag_used_minimum_radius
     "@id": "#Object.u_kronFlux_flag_used_minimum_radius"
     datatype: boolean
     description: Used the minimum radius for the Kron aperture. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1232
   - name: u_kronFlux_flag_used_psf_radius
     "@id": "#Object.u_kronFlux_flag_used_psf_radius"
     datatype: boolean
     description: Used the PSF Kron radius for the Kron aperture. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1233
   - name: u_kronRad
     "@id": "#Object.u_kronRad"
     datatype: float
     description: Kron radius (sqrt(a*b)). Measured on u-band.
-    fits:tunit:
+    fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1234
   - name: u_pixelFlags_bad
     "@id": "#Object.u_pixelFlags_bad"
     datatype: boolean
     description: Bad pixel in the Source footprint. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1235
```

Comment:
The pixel flags should be pulled out of the semialphabetical list of (mostly) flux measurements and put next to the `detect_` and `deblend_` flags.